### DISCUSSION TITLE
Capytaine pronounciation

### DISCUSSION THREAD

**Discussion by kasia129** [capytaine/capytaine#116] [General]
What is the correct pronunciation of 'Capytaine'? Is it the same as the french 'Capitaine?

**Comment by mancellin** [comment]
The name 'Capytaine' is a compound of 'Python' and the French word 'Capitaine', and I do pronounce it 'ka-pee-ten' as 'Capitaine'.
But feel free to pronounce it differently. As long as you don't pronounce it 'wha-meet'.

**Reply by kasia129** [reply]
That makes sense, thank you! I will make sure it does not sound anything like wamit.
